lib/acts_as_scrubbable/version.rb

```diff
@@ -1,3 +1,3 @@
 module ActsAsScrubbable
-  VERSION = '2.1.1'
+  VERSION = '2.1.2'
```

Comment:
I don't think we need a new version for this, since the published code hasn't changed at all. The Gemfile is only used for local development on this gem.

```suggestion
  VERSION = '2.1.1'
```

Reply:
reverted the chanes @adamstegman. main branch is now on 2.1.2